src/core/configuration.test.ts

```diff
@@ -491,3 +396,127 @@ describe("options watch-list", () => {
     });
 
 });
+
+describe("onOptionChanged", () => {
+
+    [
+        {
+            optionPath: ["option"],
+            args: { value: "new value", previousValue: "old value", component: null }
```

Comment:
```suggestion
            args: { value: "new value", prevValue: "old value", component: null }
```

Makes it shorter and still clear)